meteomatics/parsing_util.py

```diff
@@ -332,11 +332,11 @@ def parse_time_series_params(model=None, ens_select=None, cluster_select=None, i
 
 def parse_url_for_post_data(url):
     """Split the url between url and data if needed"""
-    url_splitted = url.split("/", 4)
+    url_splitted = url.split("/", 3)
     data = None
-    if len(url_splitted) > 4:
```

Comment:
The logic for if `len(url_splitted) > 4` scares me a bit, probably it was there for some reason...

I do not know if some query could potentially break when we touch that method, please make sure other types of queries are still working (all the examples are running without unexpected errors or even adding some more coverage for POST queries)

We should also add a comment why the magic number 4 or 3 was chosen, explaining the url format if necessary

Reply:
I think it was 4 because in a correct url we have two "/" after https, one "/" after the api.meteomatics, and the last "/" after the time part, but actually is a mystery why it was set to 4 and not 3 since the beginning.

Reply:
Looks fine now